concurrency/MpmcHashmap.cpp

```diff
@@ -51,14 +51,6 @@ bool MurmurHashIndex::add(uint64_t key, uint32_t value)
         }
     }
 
-    int emptySlots = 0;
-    for (auto& entry : _index)
```

Comment:
Just to clarify: it used to track being used keys, recover them and fail if nothing was recovered. But now it no recovery attempt is being made?